sim/core/stats/stats.go

```diff
@@ -426,6 +426,10 @@ type PseudoStats struct {
 	IncreasedMissChance  float64 // Insect Swarm and Scorpid Sting
 	DodgeReduction       float64 // Used by Warrior talent 'Weapon Mastery' and SWP boss auras.
 
+	IsUsingCrusaderStrikeStopAttack bool // Used to store Wild Strikes after specials
+	IsUsingJudgementStopAttack bool // Used to store Wild Strikes after specials
+	IsUsingDivineStormStopAttack bool // Used to store Wild Strikes after specials
```

Comment:
Can this be removed after the change to make them paladin options?

Reply:
removed